src/elements/metadata-list/metadata-list-item.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './metadata-list.css'
+
+import { classNames } from 'utils'
+
+const MetaDataListItem = ({
+  children,
+  label,
+  id,
+  className,
+  itemProp = null,
+  itemScope = null,
+  itemType = null,
```

Comment:
~Why don't you leave this implicit? I don't think schema.org should be part of the library even if we use it.~ I see, it's a technical issue.

I am not sure where `htmlProps` should go, to `div` or `dd` 🤔